lib/dash.coffee

```diff
@@ -55,7 +57,17 @@ plugin = module.exports =
       path = activeEditor.getPath()
       language = activeEditor.getGrammar().name
 
-    spawn('open', ['-g', @createLink(string, path, language)])
+    if platform == "win32"
+      # Exec is used because spawn escapes arguments that contain double-quotes
+      # and replaces them with backslashes. This interferes with the ability to
+      # properly create the child process in windows, since windows will barf
+      # on an ampersand that is not contained in double-quotes.
+      quotedLink = '"' + @createLink(string, path, language) + '"'
+      exec('cmd.exe /c start "" ' + quotedLink, null)
```

Comment:
Great, thanks. Good to know about what `cmd.exe /c` does. I'm still curious about the exec comment, since I don't have Windows to test. Why would we have an issue with quotes? I looked around but every example seemed to be using spawn with `spawn('cmd.exe', ['/c', ...])` or some additional escaping.